scripts/release/asset_publish.py

```diff
@@ -490,6 +504,10 @@ def _str2bool(v: str) -> bool:
         "-d", "--debug", type=_str2bool, nargs="?",
         const=True, default=False, help="debug mode",
     )
+    parser.add_argument(
+        "-va", "--validate-assets", type=_str2bool, nargs="?",
```

Comment:
This arg appears to skip validation and update of spec files. Consider renaming the flag to something like `--skip-update-spec`.